app/tests/integration/nimbus/test_experimenter.py

```diff
@@ -0,0 +1,62 @@
+import time
+
+import pytest
+from nimbus.pages.home import HomePage
+from nimbus.pages.summary import SummaryPage
+
+
+@pytest.mark.run_once
+def test_archive_experiment(
+    selenium,
+    base_url,
+    default_data,
+    create_experiment,
+):
+    selenium.get(base_url)
+    home = HomePage(selenium, base_url).wait_for_page_to_load()
+
+    # Create experiment
+    summary_page = create_experiment(selenium, home, default_data)
+
+    # Archive it on the summary page
+    summary_page.archive_button.click()
+
+    # Check it's archived on the summary page
+    time.sleep(2)
+    assert summary_page.archive_label.is_displayed()
+
+    # Check it's archived on the directory page
+    archived_tab_url = f"{base_url}?tab=archived"
+    selenium.get(archived_tab_url)
+    home = HomePage(selenium, archived_tab_url).wait_for_page_to_load()
+    archived_experiments = home.tables[0]
+    assert "Archived" in home.active_tab_text
+    for archived_experiment in archived_experiments.experiments:
+        if default_data.public_name in archived_experiment.text:
+            archived_experiment.click()
+            summary_page = SummaryPage(selenium, base_url).wait_for_page_to_load()
+            break
+
+    # Unarchive it on the summary page
+    summary_page.archive_button.click()
+
+    # Check it's unarchived on the summary page
+    time.sleep(2)
+    try:
+        assert summary_page.archive_label
+    except Exception:
```

Comment:
Does this need to catch a more specific exception? (like NoSuchElementException)

Reply:
Changed this to use the wait.until style, much nicer 👍